pkg/importer/http-datasource.go

```diff
@@ -204,58 +209,54 @@ func (hs *HTTPDataSource) Close() error {
 	return err
 }
 
-func createHTTPClient(certDir string) (*http.Client, error) {
-	client := &http.Client{
-		// Don't set timeout here, since that will be an absolute timeout, we need a relative to last progress timeout.
-	}
-
-	if certDir == "" {
-		return client, nil
-	}
+// CURL either supports a hashed-directory of certificates or a single file certificate.
+// Create a single file containing all our certificates.
+func createCAFile(certDir string) error {
+	systemCertDir := "/etc/pki/tls/certs/"
```

Comment:
Why are we hard coding some path that may or may not exist in various variations of linux. That is why we used x509.SystemCertPool, so we don't have to care about the paths of the system certs.

Reply:
I mainly want to create a file later to be used by nbdkit curl plugin, I didn't find a way to go from a certpool to a file